src/main/java/com/splunk/kafka/connect/SplunkSinkConnectorConfig.java

```diff
@@ -166,6 +168,10 @@
     static final String HEADER_SOURCETYPE_DOC = "Header to use for Splunk Header Sourcetype";
     static final String HEADER_HOST_DOC = "Header to use for Splunk Header Host";
 
+    // Load Balancer
+    static final String LB_POLL_INTERVAL_DOC = "This setting controls the load balancer polling interval. By default, "
```

Comment:
can we add clarification here and in the logs that this is an internal/ inbuilt load balancer inside the connector? (To avoid confusion if/when external LBs are used)

Reply:
It will apply even when an external load balancer is used. Its semantics remain same, so there should be no confusion.